ClinGen/clincoded
# Connects #581 variant hover preferred title

Following ticket #575, this ticket display ClinVar preferred title in curator palette and view pages.

**Changes**
- Add handling code to display preferred title in curator palette in curation central page
- Add handling code to display preferred title in family/individual/experimental view pages.

**Test Steps**
- Create GDM, add a PMID, add a new family, name Family010, enter variation ids 90973 and 9492, name proband Ind011, enter a Orphanet id, click Save. Go to curation central page and expect to see "Variants: 2" under Family010 and Ind011 in curator palette. And 2 preferred titles are shown in a pop-out window when mouse cursor is hover blue colored and underlined "2". 
<img width="495" alt="screen shot 2016-02-06 at 12 37 51 am" src="https://cloud.githubusercontent.com/assets/9206696/12865739/e6050902-cc69-11e5-9c09-3928456759d6.png">
- In addition, 2 preferred titles (or its left part with ellipses if too long) are shown in Associated Variants.
<img width="276" alt="screen shot 2016-02-06 at 12 48 26 am" src="https://cloud.githubusercontent.com/assets/9206696/12865793/7131c0aa-cc6b-11e5-9363-84e167f3a508.png">
- Click the titles, expect to redirect to variant curation page.
- Go back to curaion central page by clicking Cancel in variant curation page. View Family010, expect to see preferred titles shown as
<img width="1241" alt="screen shot 2016-02-06 at 12 53 18 am" src="https://cloud.githubusercontent.com/assets/9206696/12865806/1a18f44a-cc6c-11e5-93a4-59ad3c1472c5.png">
- Go back to curation central page and View Ind011, expect to see the same titles.
- Go back to curation central page and add a new experimental, name ExpAAA, select Rescue/Patient, cell, enter data in every required field, enter variation ids 55966, 55996, 55942 and enter "some description here" in Other description field, save. At curation central page, expect to see
<img width="308" alt="screen shot 2016-02-06 at 1 04 46 am" src="https://cloud.githubusercontent.com/assets/9206696/12865854/a5d43ebc-cc6d-11e5-8365-5b2f13ffa2a4.png">
- View ExpAAA, expect to see 4 associated variants
<img width="641" alt="screen shot 2016-02-06 at 1 06 07 am" src="https://cloud.githubusercontent.com/assets/9206696/12865860/e6ef0e68-cc6d-11e5-8959-d6ed14cb2335.png">

**End of Test**
